app/scripts/modules/kubernetes/serverGroup/configure/wizard/basicSettings.html

```diff
@@ -40,6 +40,17 @@
     </div>
   </div>
 
+  <div class="form-group">
+    <div class="col-md-3 sm-label-right">
+      TERM Grace Period
+      <help-field key="kubernetes.serverGroup.terminationGracePeriodSeconds"></help-field>
+    </div>
+    <div class="col-md-2"><input type="number" min="0"
```

Comment:
Could you please add a `seconds` input add-on here (see: https://github.com/spinnaker/deck/blob/d045ee4d01e84039dc4e3c617ce01ed9125ebcc9/app/scripts/modules/google/serverGroup/configure/wizard/autoHealingPolicy/autoHealingPolicySelector.component.html#L20-L27)?